packages/compass/src/app/index.js

```diff
@@ -375,8 +375,6 @@ app.extend({
             global.hadronApp.appRegistry.emit('refresh-data')
           );
           // as soon as dom is ready, render and set up the rest
-          const MongoDBInstance = require('mongodb-instance-model');
-          state.instance = new MongoDBInstance();
```

Comment:
Now we only create an instance when we are connected now and destroy it when we are disconnecting to try to avoid having stale instance in the app